packages/helpers/packages/ArrayHelpers/index.mdx

```diff
@@ -1,40 +1,75 @@
 ---
 name: ArrayHelpers
 route: /helpers/ArrayHelpers
-menu: 6.Helpers
+menu: Helpers
 ---
 
-# ArrayHelpers
-> ✴ Methods that helps us with arrays
-
-[![version](https://img.shields.io/npm/v/@snappmarket/array-helpers.svg?style=flat-square)](https://www.npmjs.com/package/@snappmarket/array-helpers)
-[![downloads](https://img.shields.io/npm/dm/@snappmarket/array-helpers.svg?style=flat-square)](http://www.npmtrends.com/@snappmarket/array-helpers)
-[![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](http://makeapullrequest.com)
-[![MIT License](https://img.shields.io/npm/l/@snappmarket/array-helpers.svg?style=flat-square)](https://github.com/snappmarket/frontend-toolbox/tree/master/packages/useDidUpdateEffect/blob/master/LICENSE.md)
-
-[![Watch on GitHub](https://img.shields.io/github/watchers/snappmarket/frontend-toolbox.svg?style=social)](https://github.com/snappmarket/frontend-toolbox/watchers)
-[![Star on GitHub](https://img.shields.io/github/stars/snappmarket/frontend-toolbox.svg?style=social)](https://github.com/snappmarket/frontend-toolbox/stargazers)
-
-## get started
-We provide two way of using this package `single` or `multi` :
-```bash
-npm i @snappmarket/array-helpers
-OR
-npm i @snappmarket/helpers
-```
-
-## methods
-|        |
-| ------ |
-| arrayItemAddProp                                                 |
-| flattenArray                                                 |
-| stringifyArray                                                 |
-| deepFlatten                                                 |
-
-
-## usage
-```javascript
-import { METHOD_NAME } from '@snappmarket/array-helpers';
-// or
-// import { METHOD_NAME } from '@snappmarket/helpers';
-```
+
+
+
+## Functions
+
+<dl>
+<dt><a href="#arrayItemAddProp">arrayItemAddProp(baseArray, props)</a> ⇒ array</dt>
+<dd><p>adds similar properties to children of an array</p>
+</dd>
+<dt><a href="#flattenArray">flattenArray(array, property)</a> ⇒ array</dt>
+<dd><p>makes a flat array from sub-arrays of an array</p>
+</dd>
+<dt><a href="#stringifyArray">stringifyArray(array, properties)</a> ⇒ string</dt>
+<dd><p>Create string from array</p>
+</dd>
+<dt><a href="#deepFlatten">deepFlatten(array, property, flattenArrayTemp)</a> ⇒ Array</dt>
+<dd><p>Deep flatten an array</p>
```

Comment:
The removed documentation has good information about the package, so it should not be removed, the API description could be `append` in `##API` section instead

Reply:
for better information see the difference of [PR deployed version](https://deploy-preview-149--ft-box.netlify.com/helpers/OSHelpers#functions) and [master demo](https://ft-box.netlify.com/helpers/ArrayHelpers).

Reply:
good, well i'm actually concating the generated jsdoc markdown with readme file, i should see what happend for this part, i's already supporting it.